cmd/frontend/internal/app/sign_out.go

```diff
@@ -49,6 +51,14 @@ func serveSignOutHandler(db database.DB) http.HandlerFunc {
 		}
 
 		if ssoSignOutHandler != nil {
+			http.SetCookie(w,
+				&http.Cookie{
+					Name:     signoutCookie,
+					Value:    "true",
+					HttpOnly: false,
+					Secure:   true,
+					Path:     "/",
+				})
```

Comment:
Nitpick: I'd remove the values that are set by default (`HttpOnly: false`) and intend like this:

```suggestion
			http.SetCookie(w, &http.Cookie{
				Name:   signoutCookie,
				Value:  "true",
				Secure: true,
				Path:   "/",
			})
```